### ISSUE TITLE
Allow Cheats to Run on Physical Game Cartridges

### ISSUE THREAD

**Issue by ReclaimerShawn** [DS-Homebrew/TWiLightMenu#1846]
**Is your feature request related to a problem? Please describe.**
As of right now, DS and DSi games can be ran through TWiLightMenu. AR Codes for both games can be ran as well. However, this is only for ROMs and not physical game cartridges.

**Describe the solution you'd like**
Add the ability to load Action Replay Codes on physical game cartridges, whether they be DS or DSi.

**Describe alternatives you've considered**
NitroHax3DS works for DS games but doesn't work for DSi Games. 
I could also buy an Action Replay DSi, but they go for $60+ on eBay.

**Additional context**
This could replace NitroHax3DS and Action Replay DSi, all while having the features included in TWiLightMenu.

**Comment by lifehackerhansol** [comment]
In my opinion TWiLight was not designed for this. I think this should be a NitroHax3DS request instead.

**Comment by ReclaimerShawn** [comment]
> In my opinion TWiLight was not designed for this. I think this should be a NitroHax3DS request instead.

You're correct that it wasn't made for this, but TWiLightMenu can still launch cartridges, even without cheats. Might as well add that in, as it'd be consistent with what the tool can already do for ROMs. 

Also, NitroHax3DS, with the exception of random forks and people who don't know a lot of the backend behind it, is pretty much dead from what I can tell.
